docs/image-preview.md

```diff
@@ -89,10 +89,11 @@ NVIM=1 NVIM_LOG_FILE=1 yazi
 
 ## Windows users {#windows}
 
-Currently, only the following 2 terminals support displaying images on Windows:
+Currently, only the following 3 terminals support displaying images on Windows:
 
 - WezTerm
 - [Windows Terminal Preview v1.22.2702.0](https://github.com/microsoft/terminal/releases/tag/v1.22.2702.0)
+- [Bobcat v0.9.0](https://github.com/ismail-yilmaz/Bobcat/releases/tag/0.9.0) (requires the bundled `OpenConsole.exe`)
```

Comment:
> requires the bundled `OpenConsole.exe`

Does this mean that the user needs to download `bobcat-0.9.0-win64-openconsole.zip` instead of `bobcat-0.9.0-win64.zip`? But it seems that `bobcat-0.9.0-win64-openconsole.zip` is smaller?